### ISSUE TITLE
New Files Not Available For Sharing Immediately

### ISSUE THREAD

**Issue by eatyourpeas747** [nextcloud/desktop#2073]
### Expected behaviour
I create a new file in the Nextcloud synced folder on a Windows desktop and be able to right click and be able to share it immediately.

### Actual behavior
When I put a new file in the folder  and right click it it tells me in the context menu "nextcloud resharing this file is not allowed".  If you wait about 30 seconds the share options show up.

### Steps to reproduce
1.Open an Explorer window
2. Create a new file or drag and drop and existing file into the Nextcloud sharing folder
3.immediatly right click the file to share via nextcloud

### Client configuration
Client version:2.6.4stable-Win64 (build 20200303) 

Operating system: Windows 10 1903

OS language: English

Qt version used by client package (Linux only, see also Settings dialog):

Client package (From Nextcloud or distro) (Linux only):

Installation path of client: C:\Program Files (x86)\Nextcloud

### Server configuration

Nextcloud version: Nextcloud 18.0.4

**Comment by Whissi** [comment]
I have the same problem:

![image](https://user-images.githubusercontent.com/729436/97707883-3d51ae80-1ab8-11eb-9619-5f53482a53ec.png)

Problem is present in >=nextcloud-client-2.6. I have to downgrade to 2.5.x to avoid this problem.

**Comment by Whissi** [comment]
Still an issue.

**Comment by Orhideous** [comment]
Also reproducible on Ubuntu 20.40 since at least 3.x client.

**Comment by Whissi** [comment]
Not stale.

**Comment by Whissi** [comment]
This bug is not stale.

**Comment by Rello** [comment]
Hello,
this is related to the sync. For a share, the clients checks the server for the metadata and authorisations. 
your newly created file was not synced to the server yet due to e.g. server performance. there is no way around to rely on the server confirming the file

**Comment by Whissi** [comment]
@Rello This is wrong. As you can see from my screenshot, I am experiencing the problem for already sync'ed files. Also, why does it work in previous client versions if you are right that the problem is that the server cannot yet confirm the file? That doesn't make sense... please re-open this bug.
